plugins/proclist/Pclass.cpp

```diff
@@ -30,7 +30,11 @@ class StrTok
 		tok(token)
 	{
 		buf.reset(wcsdup(str));
+#if defined(_UCRT)
```

Comment:
Is there any reason to not use `std::wcstok(buf.get(), token, &pt);` unconditionally?

Reply:
I won't work for mingw-msvcrt:
https://github.com/mingw-w64/mingw-w64/blob/e0d90587c9825614fe59eda9066ad12fe72ff9bf/mingw-w64-headers/crt/string.h#L147-L151